packages/admin/blocks-admin/src/iframebridge/IFrameMessage.ts

```diff
@@ -1,11 +1,20 @@
 // Same file in admin and site
 
 // Messages sent from iFrame -> Admin
+import { ExternalLinkBlockData } from "../blocks.generated";
 
 export enum IFrameMessageType {
     Ready = "Ready",
     SelectComponent = "SelectComponent",
     HoverComponent = "HoverComponent",
+    /**
+     * @deprecated Use SitePreviewIFrameMessageType.OpenLink instead
+     */
+    OpenLink = "OpenLink",
```

Comment:
why did you re-add this? Simply to be compatible?

Reply:
Yes. I simply reviewed the original PR and looked for removed parts. We probably don't need this.

Reply:
If we revert this commit in next I'm ok. 

Next time we should target next branch also for larger refactorings

Reply:
We can't simply revert the commit, as we should keep the `sendSitePreviewIFrameMessage` and `SitePreviewIFrameMessageType` exports. We may remove the deprecated parts however.